src/specimens/Table.js

```diff
@@ -40,13 +40,25 @@ function getStyle(theme) {
   };
 }
 
-const Cell = (value, id, style, heading) => {
-  const content = typeof value === 'string'
-    ? renderMarkdown({text: value})
-    : value || <p style={{opacity: 0.2}}>—</p>;
+const Cell = ({value, style, heading}) => {
+  let content;
+  if (typeof value === 'string') {
+    content = renderMarkdown({text: value.toString()});
+  } else if (value === void 0) {
```

Comment:
I don't really know 😄. `void 0` is safer than `undefined` (in old environments which let you redefine `undefined`) but `typeof foo === 'undefined'` guards against undeclared vars. Seems like an issue for https://github.com/interactivethings/eslint-config-interactivethings. I'm going to leave it as it is, and we can fix it later.